File: NFCDirectionalTracking.Core/Models/ProductDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NFCDirectionalTracking.Core.Models
{
    public class ProductDetails : ProductListing
    {
        public int ColourID { get; set; }

        public string Colour { get; set; }

        public int SizeID { get; set; }

        public string Size { get; set; }
    }
}

File: NFCDirectionalTracking/script/src/models/ProductDetails.ts
module NFCApp {
    export class ProductDetails {
        ID: number;
        ColourID: number;
        SizeID: number;
        Name: string;
        Colour: string;
        Size: string;
        constructor(data: any) {
            this.ID = data.ID;
            this.ColourID = data.ColourID;
            this.SizeID = data.SizeID;
            this.Name = data.Name;
            this.Colour = data.Colour;
            this.Size = data.Size;
        }
    }
} 